cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -1,79 +1,54 @@
-import { fieldHelpPreview, labelByPosition, labelWidthSliderByName } from '../../locators';
+import { labelWidthSliderByName, labelByPositionNoIFrame } from '../../locators';
 import {
-  radioButtonByPosition,
   radioButtonFieldset,
   radioButtonLegend,
   radioButtonGroupNoIframe,
   radioButtonComponentByPositionNoIframe,
 } from '../../locators/radioButton/index';
 import { setSlidebar, positionOfElement } from '../helper';
-
-const INLINE = 'carbon-radio-button__help-text--inline';
-const RADIOBUTTON_INPUT_CSS = 'div:nth-child(1) input';
-
-Then('fieldHelpInline is enabled', () => {
-  fieldHelpPreview().should('have.class', INLINE);
-});
-
-Then('fieldHelpInline is disabled', () => {
-  fieldHelpPreview().should('not.have.class', INLINE);
-});
+import { FIELD_HELP_PREVIEW } from '../../locators/locators';
 
 Then('{string} radioButton on preview is {word}', (position, text) => {
-  labelByPosition(positionOfElement(position)).should('have.text', text);
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.text', text);
 });
 
-Then('{string} RadioButton has value {word}', (radioButtonName, text) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'value').should('contain', text);
+Then('{string} RadioButton has value {word}', (position, text) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.attr', 'value').should('contain', text);
```

Comment:
We should move 
radioButtonComponentByPositionNoIframe(index).find('input') this to the `index.js` file and use here only the 
`radioButtonComponentByPositionNoIframe(positionOfElement(position))`

Reply:
fixed